txgh/lib/txgh/gitlab_api.rb

```diff
@@ -0,0 +1,75 @@
+require 'base64'
+require 'gitlab'
+
+module Txgh
+  class GitlabApi
+    class << self
+      def create_from_credentials(_login, access_token, repo_name)
+        Gitlab.endpoint = 'https://gitlab.com/api/v4'
```

Comment:
Any way to configure the URL at the instance level instead of globally? Also, can we move the URL into a constant?